backend/s3/options_test.go

```diff
@@ -81,6 +81,21 @@ func (o *optionsTestSuite) TestGetClient() {
 				o.NotNil(client.Options().Credentials, "credentials are set")
 			},
 		},
+		{
+			name: "role ARN set using key and secret passed in options",
+			opts: Options{
+				AccessKeyID:     "AKIAIOSFODNN7EXAMPLE",
+				SecretAccessKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
+				Region:          "some-region",
+				RoleARN:         "arn:aws:iam::123456789012:role/my-role",
+			},
+			expected: func(o *optionsTestSuite, client *s3.Client, err error) {
+				o.Require().NoError(err)
+				o.NotNil(client, "client is set")
+				o.Equal("some-region", client.Options().Region, "region is set")
+				o.NotNil(client.Options().Credentials, "credentials are set")
+			},
+		},
```

Comment:
These tests (the preexisting ones this is modeled after) really aren't great, are they? I mean we just know that the there are credentials but not necessarily that they are correct.  🤷‍♂️